dlf/plugins/pageview/tx_dlf_utils.js

```diff
@@ -561,38 +561,48 @@ dlfUtils.isCorsEnabled = function(imageObjs) {
     // fetch data from server
     // with access control allowed
     let response = true;
-
+    const type = 'HEAD';
     imageObjs.forEach(function(imageObj) {
         let url = imageObj.mimetype === dlfUtils.CUSTOM_MIMETYPE.ZOOMIFY
-            ? imageObj.url.replace('ImageProperties.xml', 'TileGroup0/0-0-0.jpg')
-            :
-            imageObj.mimetype === dlfUtils.CUSTOM_MIMETYPE.IIIF
-                ? dlfViewerSource.IIIF.getMetdadataURL(imageObj.url)
+            ? dlfUtils.buildXhrURI(imageObj.url.replace('ImageProperties.xml', 'TileGroup0/0-0-0.jpg'), true)
+            : imageObj.mimetype === dlfUtils.CUSTOM_MIMETYPE.IIIF
+                ? dlfUtils.buildXhrURI(dlfViewerSource.IIIF.getMetdadataURL(imageObj.url), false)
```

Comment:
Thank you for your investigation.

Indeed our server currently supports only `Access-Control-Allow-Methods GET`. This should be no law but it seems to be a setting which is commonly used (if you can say "commonly" in case of CORS anyway).

I'm not happy with this construction with server side PHP prefetching the header. And I agree, it should be done in browser context. But it must work in `mixed_content` conditions too.

For now, I would suggest to merge this patch and investigate this further. Maybe you want to open an issue?